daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf2.proto

```diff
@@ -302,23 +269,14 @@ message Type {
     int32 interned = 13; // *Available in versions >= 1.11*
   }
 
-  reserved 4; // This was fun.  Removed in favour of PrimType.ARROW
-  reserved 6; // This was list. Removed in favour of PrimType.LIST
-  reserved 8; // This was contract_id. Removed in favour of PrimType.CONTRACT_ID
-  reserved 9; // This was update. Removed in favour of PrimType.UPDATE
-  reserved 10; // This was scenario. Removed in favor of PrimType.SCENARIO
-
 }
 
-// Primitive constructors
-enum PrimCon {
-
+// Builtinitive constructors
```

Comment:
search-and-replace leads to particitrousers :) there's handful more of these

```suggestion
// Builtin constructors
```

Reply:
:bow: